src/data/faq_de.json

```diff
@@ -100,15 +100,30 @@
-                            "Wenn Sie den QR Code bereits gescannt haben, kann dieser leider nicht ein zweites Mal gescannt werden, ebenso ist das Löschen eines zuvor eingegebenen QR Codes und erneutes einscannen nicht möglich. Wir müssen sicherstellen, dass nur ein Telefon den QR Code scannen und seine Diagnoseschlüssel übermitteln kann. Aus Datenschutzgründen können wir die ID des Telefons nicht speichern, deshalb ist der QR Code nur für einen Scan gültig.",
-                            "Wir prüfen gerade, wie die Verbesserung der Benutzerschnittstelle der App verbessert werden kann, um nicht funktionierende QR-Codes zu erkennen und Sie darüber informieren zu können. Weitere Details zu den Fehlern finden Sie <a href='https://github.com/corona-warn-app/cwa-documentation/issues/400' target='_blank' rel='noopener noreferrer'>auf GitHub</a>."
+                            "Sofern Sie nach einem Corona-Test Ihren QR-Code in die App eingescannt haben und innerhalb von wenigen Tagen kein Testergebnis über die App mitgeteilt bekommen, könnte es sein, dass das entsprechende Labor noch nicht vollständig mit der Corona-Warn-App zusammenarbeitet oder das Feld 9 'Einverständnis des Versicherten zum Übermitteln des Testergebnisses für Zwecke der Corona-Warn-App' im Formular nicht markiert wurde.",
+                            "Erkundigen Sie sich daher bitte, ob das Labor, mit dem der Arzt oder das Testcenter zusammenarbeiten an die Corona-Warn-App angebunden ist und Feld 9 auf dem Formular angekreuzt ist. Nur dann ist sichergestellt, dass das genutzte Verfahren mit der Corona-Warn-App funktioniert.",
+                            "Derzeit sind 90% der vorhandenen Testkapazitäten in Deutschland durch den kompletten digitalen Prozess abgedeckt. Wir arbeiten mit den Laboren weiter daran, die Lücken zu schließen und alle Labore für den Prozess anzubinden und aufzurüsten. Deshalb kann es vorkommen, dass ein Labor zwar die Beauftragung für den Test mit dem richtigen Formular bekommt, das Testergebnis aber auf einem anderen Weg an die getestete Person übermitteln muss.",
+                            "Rufen Sie bitte beim zuständigen ärztlichen Fachpersonal bzw. Testcenter/Gesundheitsamt an, um Ihr Testergebnis zu erhalten. Wird Ihnen dabei ein positives Testergebnis mitgeteilt, können Sie über die Hotline unter der +49 800 7540002 eine TeleTAN beantragen, um über die Corona-Warn-App Ihre Risikobegegnungen zu warnen. Tragen Sie für diesen Zweck die erhaltene TeleTAN unter",
+                            "<ul><li>'Nächste Schritte' -> 'Weiter' -> 'TAN-Eingabe'</li></ul>",
```

Comment:
Change `"<ul><li>'Nächste Schritte' -> 'Weiter' -> 'TAN-Eingabe'</li></ul>",` to
`"<ul><li>'NÄCHSTE SCHRITTE' -> 'TAN-Eingabe'</li></ul>",`

Screenshots from released CWA Android 1.9.1
---
![De_wurden_Sie_gestestet](https://user-images.githubusercontent.com/66998419/102688933-aa510d00-41fa-11eb-9b17-ce09e917af7a.jpg)

---
![De_Ihr_Test_ist_positiv](https://user-images.githubusercontent.com/66998419/102688937-b046ee00-41fa-11eb-8c06-9ad20f554397.jpg)